openedx/features/enterprise_support/admin/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+
+class CSVImportForm(forms.Form):
+    csv_file = forms.FileField(label='CSV File')
```

Comment:
can you update the label to something more descriptive? like "CSV file containing enrollment attributes" and also provide `help_text` argument with proper text?